frontend/front/src/features/breadcrumb/breadcrumbSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const breadcrumbSlice = createSlice({
+  name: "breadcrumbs",
+  initialState: {
+    links: [],
+  },
+  reducers: {
+    pushBreadcrumb: (state, action) => {
```

Comment:
Handling this with redux seems overcomplicated to me, I'm interested what other's think.